phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Beans">
+    <description>The code standard for Beans.</description>
+    <file>.</file>
+    <exclude-pattern>dist/*</exclude-pattern>
```

Comment:
Add a new empty line and comment above this line of code, as we want to make this file more readable for all contributors:

```

    <!-- Ignore these folders -->
    <exclude-pattern>dist/*</exclude-pattern>
    <exclude-pattern>node_modules/*</exclude-pattern>
    <exclude-pattern>vendor/*</exclude-pattern>
```